tools/eslint/base_rules.js

```diff
@@ -0,0 +1,164 @@
+module.exports = {
```

Comment:
Just as in the `packs-sdk` repo, the idea here is to be able to copy over our Coda-wide rules from the `coda` repo, and have our overrides apply the remaining packs-specific styles, rather than having to have to figure out how to merge in new rules from the `coda` side that we want to incorporate later.